packages/ChangesReloaded-Core.package/CRBrowser.class/instance/updateLoadButton.st

```diff
@@ -0,0 +1,6 @@
+updating
+updateLoadButton
+
+	self 
+		changed: #loadButtonLabel;
+		changed: #isLoadButtonEnabled
```

Comment:
Aaha so this is why we need these extra methods. Hmm I'm still wondering if this extra layer of abstraction is really neccessary tho. Or is this even deeper ingrained in all the squeaky things?

Reply:
The buttons are built on initialization. If we set the button to a fix boolean there, we'd have to retrieve the button morph later to set enable/disable state. You don't want to access the button morph at runtime.
That's why, instead of a fix boolean, you put this method into the button specification. The button morph then evaluates the method at runtime and gets enabled/disabled accordingly. That's why we need this function